commerce_coordinator/apps/commercetools/tasks.py

```diff
@@ -60,17 +62,17 @@ def refund_from_stripe_task(
     try:
         payment = client.get_payment_by_key(payment_intent_id)
         if has_full_refund_transaction(payment):
-            logger.info(f"Stripe charge.refunded event received, but Payment with ID {payment.id} "
+            logger.info(f"[refund_from_stripe_task] event received, but Payment with ID {payment.id} "
                         f"already has a full refund. Skipping task to add refund transaction")
             return None
         updated_payment = client.create_return_payment_transaction(
             payment_id=payment.id,
             payment_version=payment.version,
             stripe_refund=stripe_refund
         )
+        logger.info(f"[refund_from_stripe_task] Successfully created refund transaction for Payment {payment.id}.")
         return updated_payment
     except CommercetoolsError as err:
-        logger.error(f"Unable to create refund transaction for payment [ {payment.id} ] "
-                     f"on Stripe refund {stripe_refund.id} "
-                     f"with error {err.errors} and correlation id {err.correlation_id}")
+        logger.error(f"[refund_from_stripe_task] Failed to create refund transaction for Payment {payment.id}. "
+                     f"Stripe Refund ID: {stripe_refund.id}. Error: {err.errors}, Correlation ID: {err.correlation_id}")
```

Comment:
I think we should keep it in the form of a sentence like before. We use similar way of logging in customer-twou repo as well